website/_includes/navigation.html

```diff
@@ -8,5 +8,5 @@
     {% endif %}
   </li>
 {% endfor %}
-<li><a href="https://github.com/openrubyfoundation/rubygalaxy"><button>Submit a Talk</button></a></li>
+<li><a href="https://www.papercall.io/ruby-galaxy"><button>Submit a Talk</button></a></li>
```

Comment:
Now all `Submit a Talk` links and buttons send the user to Paper Call instead of our repo which was the initial design.